test/e2e/opentelemetry/instrumentation/opentelemetry.test.ts

```diff
@@ -18,6 +18,7 @@ describe('opentelemetry', () => {
     dependencies: require('./package.json').dependencies,
     env: {
       TEST_OTEL_COLLECTOR_PORT: String(COLLECTOR_PORT),
+      NEXT_TELEMETRY_DISABLED: '1',
```

Comment:
This is apparently set in CI. Without this, running this locally (with Webpack) fails.